tools/dev-server/index.bzl

```diff
@@ -0,0 +1,125 @@
+load("@build_bazel_rules_nodejs//internal/common:sources_aspect.bzl", "sources_aspect")
+
+"""
+  Gets the workspace name of the given rule context.
+"""
+
+def _get_workspace_name(ctx):
+    if ctx.label.workspace_root:
+        # We need the workspace_name for the target being visited.
+        # Skylark doesn't have this - instead they have a workspace_root
```

Comment:
"Starlark" now

Reply:
Good catch 💯 I copied this over from [here](https://github.com/bazelbuild/rules_nodejs/blob/master/packages/typescript/src/internal/devserver/ts_devserver.bzl#L49)